uspace/app/init/init.c

```diff
@@ -207,7 +207,7 @@ static int srv_startl(const char *path, ...)
 		printf("%s: Server %s failed to start (exit code %d)\n", NAME,
 		    path, retval);
 	
-	return retval;
+	return retval == 0 ? EOK : EPARTY;
```

Comment:
EPARTY was meant as a special error code returned only from the kernel so that it is possible to distinguish one particular error condition from whatever the recipient may return to the sender. I know it is not enforced now other than by convention, but I think that we should not proliferate EPARTY to other places.

Reply:
Ah, good to know. Which error code do you think would fit best for the case of "child task exited unsuccessfully"?

Reply:
I solved this by introducing the IPC_CALL_AUTO_REPLY flag which can only be set by the kernel.